vars/neoDeploy.groovy

```diff
@@ -10,21 +11,73 @@ import static com.sap.piper.Prerequisites.checkScript
 
 @Field String STEP_NAME = getClass().getName()
 @Field Set GENERAL_CONFIG_KEYS = [
-    'neo'
+    'neo',
+    /**
+     * The Jenkins credentials containing user and password used for SAP CP deployment.
+     * @parentConfigKey neo
+     */
+    'credentialsId',
+    /**
+     * Name of the application you want to manage, configure, or deploy.
+     * @parentConfigKey neo
+     */
+    'application',
+    /**
+     * Name of SAP Cloud Platform application runtime.
+     * @parentConfigKey neo
+     */
+    'runtime',
+    /**
+     * Version of SAP Cloud Platform application runtime.
+     * @parentConfigKey neo
+     */
+    'runtimeVersion',
+    /**
+     * The SAP Cloud Platform host to deploy to.
+     * @parentConfigKey neo
+     */
+    'host',
+    /**
+     * The SAP Cloud Platform account to deploy to.
+     * @parentConfigKey neo
+     */
+    'account'
 ]
 @Field Set STEP_CONFIG_KEYS = GENERAL_CONFIG_KEYS.plus([
+    /** @see dockerExecute */
     'dockerEnvVars',
+    /** @see dockerExecute */
     'dockerImage',
+    /** @see dockerExecute */
     'dockerOptions',
+    /**
```

Comment:
To my understanding `neoHome` is not supported anymore. This parameter should not be present here. There should also be no documentation for that parameter. I vote for removing this.

Reply:
Done